include/config/general.h

```diff
@@ -72,6 +72,9 @@
 #define SUMMARY_SCREEN_NATURE_COLORS TRUE    // If TRUE, nature-based stat boosts and reductions will be red and blue in the summary screen.
 #define HQ_RANDOM                    TRUE    // If TRUE, replaces the default RNG with an implementation of SFC32 RNG. May break code that relies on RNG.
 #define COMPETITIVE_PARTY_SYNTAX     TRUE    // If TRUE, parties are defined in "competitive syntax".
+#define AUTO_SCROLL_TEXT             FALSE   // If TRUE, text will automatically scroll to the next line after NUM_FRAMES_AUTO_SCROLL_DELAY. Players can still press A_BUTTON or B_BUTTON to scroll on their own.
```

Comment:
Considering we're hijacking the Union Room's autoscroll, are we sure that now being able to use the A and B buttons in the Union Room to "override" the autoscroll doesn't break Union Room functionality in some shape or form? I'd personally err on the side of caution and prevent overrides in the Union Room to preserve that functionality as much as possible.

Feel free to say this is out of scope, but I'd imagine that rather than a global config this may actually be used more if it's flag-based so people can turn it on and off depending on the in-game context. Maybe the delay could be variable-controlled too (but again, feel free to say "not now")

Reply:
> Considering we're hijacking the Union Room's autoscroll, are we sure that now being able to use the A and B buttons in the Union Room to "override" the autoscroll doesn't break Union Room functionality in some shape or form? 

It could. I can add another check so that when `gTextFlags.autoScroll` is set ,the A and B button don't work.

> Feel free to say this is out of scope, but I'd imagine that rather than a global config this may actually be used more if it's flag-based so people can turn it on and off depending on the in-game context. Maybe the delay could be variable-controlled too (but again, feel free to say "not now")

I mean that's what `gTextFlags.autoScroll` is, which is already in vanilla. If you want to preserve your previous comment and this one, it sounds like you are suggesting:

1) Remove the global config
2) Introduce `gTextFlags.autoScrollNotMultiplayer` which triggers this functionality and/or allows for the use of A / B button
3) Introduce scripting commands to toggle this new flag

correct?

Reply:
@Bassoonian anything left here?